pkg/requesthandler/commitments.go

```diff
@@ -38,7 +38,7 @@ func (r *RequestHandler) GetCommitmentByID(commitmentID iotago.CommitmentID) (*m
 
 	commitment, err := r.protocol.Engines.Main.Get().Storage.Commitments().Load(commitmentID.Slot())
 	if err != nil {
-		return nil, ierrors.Wrapf(echo.ErrInternalServerError, "failed to load commitment, commitmentID: %s, slot: %d, error: %w", commitmentID, commitmentID.Slot(), err)
+		return nil, ierrors.Wrapf(echo.ErrInternalServerError, "failed to load commitment, commitmentID: %s, slot: %d, error: %v", commitmentID, commitmentID.Slot(), err)
```

Comment:
I don't understand the change from `%w` to `%v`.
With that change, the `ierrors` lib is unable to properly wrap and chain the errors.

Can you revert these changes?

In this special case you even could do something like this to be more clean:
```suggestion
		return nil, ierrors.Join(echo.ErrInternalServerError, ierrors.Wrapf(err, "failed to load commitment, commitmentID: %s, slot: %d", commitmentID, commitmentID.Slot()))
```